test/runtests.jl

```diff
@@ -1,10 +1,14 @@
 using IFTPipeline
```

Comment:
Does line number 2 replace this first line now?

Reply:
L1 brings into scope stuff from IFT and L2 brings into scope unexported names in IFTPipeline that I use to kind of mimic the CLI behavior within the test suite.